scale_build/image/iso.py

```diff
@@ -0,0 +1,103 @@
+import glob
+import itertools
+import os
+import shutil
+
+from scale_build.config import VERSION
+from scale_build.utils.manifest import get_manifest
+from scale_build.utils.run import run
+from scale_build.utils.paths import BUILDER_DIR, CD_DIR, CHROOT_BASEDIR, CONF_GRUB, RELEASE_DIR, TMP_DIR
+
+from .logger import get_logger
+from .manifest import UPDATE_FILE
+from .utils import run_in_chroot
+
+
+def install_iso_packages():
+    installer_logger = get_logger('cdrom-packages')
+    run_in_chroot('apt update', logger=installer_logger)
+
+    # echo "/dev/disk/by-label/TRUENAS / iso9660 loop 0 0" > ${CHROOT_BASEDIR}/etc/fstab
+    for package in get_manifest()['iso-packages']:
+        run_in_chroot(
+            f'apt install -y {package}', logger=installer_logger, exception_message=f'Failed apt install {package}'
+        )
+
+    os.makedirs(os.path.join(CHROOT_BASEDIR, 'boot/grub'), exist_ok=True)
+    shutil.copy(CONF_GRUB, os.path.join(CHROOT_BASEDIR, 'boot/grub/grub.cfg'))
+
+
+def make_iso_file():
+    iso_logger = get_logger('cdrom-iso')
+    for f in glob.glob(os.path.join(RELEASE_DIR, '*.iso*')):
+        os.unlink(f)
+
+    # Set default PW to root
+    run_in_chroot(r'echo -e \"root\nroot\" | passwd root', logger=iso_logger)
+
+    # Create /etc/version
+    with open(os.path.join(CHROOT_BASEDIR, 'etc/version'), 'w') as f:
+        f.write(VERSION)
+
+    # Copy the CD files
+    for source, destination in (
```

Comment:
Can't we just copy all `cd-files` we have without listing them here?

Reply:
They have specific source / destination, please advise if you think we should be perhaps re-arranging cd folder so that we can copy it directly in the chroot ?

Reply:
Yeah, I think that'll be easier to use in the future